src/main/java/seedu/scheduler/model/ModelManager.java

```diff
@@ -521,11 +523,13 @@ public void updateScheduleList() {
      */
     @Override
     public List<Schedule> getSchedulesList() {
+        Collections.sort(schedulesList);
         return schedulesList;
     }
 
     @Override
     public List<List<String>> getTitlesLists() {
+        Collections.sort(schedulesList);
         List<List<String>> titlesLists = new LinkedList<>();
         for (Schedule schedule : schedulesList) {
```

Comment:
Actually can just do Collections.sort once in the setSchedules() method, instead of 3 times over here.